packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -656,13 +659,20 @@ const initialize = async () => {
 		optimizedInterventionPolicy.value = await getInterventionPolicyById(optimizedPolicyId);
 	}
 
-	modelStateAndObsOptions.value = model.value?.model.states.map((state: any) => state.id);
-
-	/** Until supported by pyciemss-service, do not show observables.
-	if (model?.semantics?.ode.observables) {
-		modelStateAndObsOptions.value.push(...model.semantics.ode.observables.map((observable: any) => observable.id));
+	modelStateAndObsOptions.value = model.value?.model.states.map((state: any) => ({
+		label: state.id,
+		value: `${state.id}_state`
+	}));
+
+	// Add obs:
+	if (model.value?.semantics?.ode.observables) {
+		modelStateAndObsOptions.value.push(
+			...model.value.semantics.ode.observables.map((observable: any) => ({
+				label: observable.id,
+				value: `${observable.id}_observable`
+			}))
```

Comment:
Are these the field-names used in pyciemsss-service (label / value) ? Wondering if we can be a bit more explicit than just saying "value".

Would be a bit easier to read if you extract out
- model.value?.semantics?.ode, and
- model.value?.model

Reply:
I will extract these out

Label = dropdown label for UI
value = the value being used for the pyciemss-service call

Maybe I can rename value to something more relating to constraint target variable